contact-us.md

```diff
@@ -1,4 +1,4 @@
-# ✍ Contact Us
+# <span class="emoji-bullet">✍</span> Contact Us
```

Comment:
Just noticed an existing formatting issue on Line 26 (it won't let me highlight):

![Screenshot from 2021-11-03 10-50-53](https://user-images.githubusercontent.com/16181459/140083653-7eb9679e-915c-4dce-a026-cbf997b4220c.png)